shell/platform/android/io/flutter/view/ResourceExtractor.java

```diff
@@ -33,7 +33,8 @@
 
     @SuppressWarnings("deprecation")
     static long getVersionCode(PackageInfo packageInfo) {
-        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.P) {
+        // Linter needs 28 hardcoded
```

Comment:
Seems strange, but can we document that 28 is P?

Reply:
Is the number 28 not available as a constant?  I thought all SDK numbers were available.

Reply:
It is a constant, `Build.VERSION_CODES.P`. For some reason the linter consistently fails all checks using the constant though and passes when I'm using just the raw int `28`. I'll try to fiddle with it some more and document it if I can't get a fix.